src/tests/get_released_revisions.py

```diff
@@ -84,10 +84,13 @@ def get_image_name_in_registry(img_name: str, revision: str) -> str:
             releases = json.load(rf)
 
         released_revisions[img] = []
-        for risks in releases.values():
+        for revision, risks in releases.items():
```

Comment:
```suggestion
        for track, risks in releases.items():
```

it's the track, not the revision (same below in the logs)